src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2149,7 +2167,9 @@ export default class Aircraft {
                     }
                 }
             }
-        } else {
+        }
+
+        else {
```

Comment:
make sure an else if is on the same line as the closing brace for the previous conditional.